cmd/extract.go

```diff
@@ -17,6 +17,7 @@ package cmd
 import (
 	"fmt"
 	"os"
+	s "strings"
```

Comment:
I would prefer not to use `s` and instead use `strings` to make the code clearer.